src/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Button ({product, amount, clickHandlerAdd, clickHandlerSubtract, disabled, className}) {

```

Comment:
Ik zou in het vervolg een andere naam kiezen voor dit component. In feite is het niet echt een button op zichzelf